RELEASE-NOTES.md

```diff
@@ -1,5 +1,32 @@
 # Prefect Release Notes
 
+## Release 2.7.10
+
+### Enhancements
```

Comment:
It looks like we're missing all the cancellation related work?

Reply:
Yes, it's in my description as in progress :)